### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by kkchu791** [PythonHollywood/pet-bot#2]
Write a brief summary of what this app is supposed to do and how to run the app.

**Comment by belteshassar** [comment]
I did a small update. Should be fleshed out more. 

f30600322eab601243c416ea3e99573f3119dd6b

**Comment by oitkjf** [comment]
Yeah, we'll need to clarify these issues as a group. I think this week's venue will be more conducive to group discussion.
